frontend/public/components/operator-lifecycle-manager/catalog-source.tsx

```diff
@@ -58,6 +62,11 @@ export const CatalogSourceDetailsPage: React.SFC<CatalogSourceDetailsPageProps>
     isList: true,
     namespace: props.match.params.ns,
     prop: 'subscriptions',
+  }, {
```

Comment:
Add `OperatorGroups` to the `<Firehose>` so that `requireOperatorGroups` HOC will show "Not Enabled" message.